### PR TITLE
Enabling integration tests on WF@JDK9; WildFly Arquillian update

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://hibernate.atlassian.net/browse/HV-1128
https://hibernate.atlassian.net/browse/HV-1129

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@Sanne As you asked me: you may try the same in HSEARCH. Just make sure to use a recent JDK 9 (ideally, the latest, b139). Earlier ones expose a bug in conjunction with Weld.

**Comment by Sanne** [comment]
@gunnarmorling ah, nice! I was aware of the Weld issue, didn't know it was fixed in the JDK. Thanks!

File: pom.xml [hunk 1]
```diff
@@ -896,6 +896,9 @@
             <properties>
                 <surefire.argLine.extension>--add-modules java.xml.bind</surefire.argLine.extension>
             </properties>
+            <modules>
+                <module>integration</module>
+            </modules>
```
Comment (gsmet) [inline pom.xml]: Wouldn't it make more sense to add back the module to the global list?

**Comment by gsmet** [comment]
Merged! Thanks!

**Pull request merged**